dev/playbooks/README.md

```diff
@@ -1,11 +1,22 @@
 SatelliteQE Dev Environment Playbooks
 =====================================
 
-The ansible playbooks to deploy Fedora 23 development machine
+The ansible playbooks to deploy Fedora 23/24 development machine
 
 install
 --------------
 
+### auto
+
+```bash
+wget -O /tmp/deploy_my_machine.sh https://raw.githubusercontent.com/SatelliteQE/sat6helper/master/dev/playbooks/deploy_my_machine.sh && . /tmp/deploy_my_machine.sh
```

Comment:
You can make this one liner install to not require writing to a local file like this https://gist.github.com/elyezer/d1b011f1f5ac66b35b0f#gistcomment-1725944